README.md

```diff
@@ -1,3 +1,64 @@
-# hapi-login [ *Coming Soon* !!]
+<<<<<<< HEAD
+# Hapi Login
 
-A Complete Login Solution for Hapi.js Apps/APIs
+A ***Complete Login Solution*** for **Hapi.js** Apps/APIs to get you
+***up and running*** in the next ***5 minutes***.
+
+## Why?
+
+Letting people login to your App/API is *essential* any time
+there is personalized content/functionality to display.
+
+We needed a *easy* way of doing Login/Authentication for our projects
+that we could drop into any project and be up-and running in minutes
+and thus avoid people re-inventing the wheel too often.
+
+After much research and investigation, we decided to use a few *existing*
+**Hapi** modules together to form a re-useable starter package.
+
+## What?
+
+Login for Hapi Apps/APIs which gives you a set of routes
+and a predictable usage pattern.
+
+### Tested
+
+Our *objective* is to *extensively* test every aspect of this package
+so that we can *rely* on the package for our *high-traffic/security* projects.
+
+If you spot any area for improvement, please create an issue:
+https://github.com/dwyl/hapi-login/issues so we can discuss!
+
+## How?
+
+As the name suggests, this plugin is built for apps built with the  [Hapi.js](https://github.com/nelsonic/learn-hapi) web framework.  
+If you or *anyone* on your team are new to Hapi, we
+have an **introductory tutorial**: https://github.com/nelsonic/learn-hapi
+
+
+
+
+### Dependencies
```

Comment:
@jalcine we expect the end result to be pretty useful to the community.
Authentications will be enabled by the presence of the environment variables required to power those schemes. Watch this space. :wink: